### PR TITLE
[IMP] website: "pages" page

### PR DESCRIPTION
**Pull request by toaa-odoo**
Adding a documentation on "pages" within the already existing "page" toctree

task-3383323

@auva-odoo Hi Audrey, could you have a - thorough - look at this PR? I wrote it based on your notes, but I could not understand everything. You'd know better what is supposed to be in here :)

### CODE REVIEW

File: content/applications/websites/website/pages.rst [hunk 1]
```diff
@@ -1,9 +1,186 @@
...
+your site to a new :doc:`domain </administration/maintain/domain_names>`. It can also be used to
+improve :doc:`pages/seo`.
+
+To access existing URL redirections for your website and create new ones, :doc:`activate the
+developer mode </applications/general/developer_mode>` and go to :menuselection:`Website -->
+Configuration --> Redirects`.
+
+.. note::
+   A record is added automatically everytime you :ref:`modify a page's URL <website/page_properties>`
+   and enable :guilabel:`Redirect Old URL`.
```
Comment (LoredanaLrpz) [inline content/applications/websites/website/pages.rst:146]: You can seperate at any "space". Here's the only suggestion for which I checked the number of characters x) 
```suggestion
.. note::
   A record is added automatically everytime you :ref:`modify a page's URL
   <website/page_properties>` and enable :guilabel:`Redirect Old URL`.
```

File: content/applications/websites/website/pages.rst [hunk 2]
```diff
@@ -1,9 +1,186 @@
...
+- :guilabel:`Sequence`: to define the order in which redirections are performed, e.g. in the case of
+  redirect chains (i.e., a series of redirects where one URL is redirected to another one, which is
+  itself further redirected to another URL).
+
+Toggle the :guilabel:`Activate` switch to deactivate the redirection.
+
+.. note::
+   - 301 and 308 redirections are cached by browsers.
+   - 404, 301 and 302 redirections only work if the original page has been :ref:`unpublished <website/un-publish-page>`
+     or :ref:`deleted <website/delete-page>`.
```
Comment (LoredanaLrpz) [inline content/applications/websites/website/pages.rst:178]: Same as above :)
```suggestion
   - 404, 301 and 302 redirections only work if the original page has been :ref:`unpublished
     <website/un-publish-page>` or :ref:`deleted <website/delete-page>`.
```

**Review by LoredanaLrpz — CHANGES_REQUESTED** [review]
Hey, @auva-odoo , great work overall =D
Here's my review 
_Please note that I usually don't check the count of characters when I make suggestions x)_

Also, don't forget to change the commit msg since you co-authored the PR 
--> commit --amend and write
`Co-authored-by: Toaa <toaa@odoo.com>`
`Co-authored-by: Auva <auva@odoo.com>`

File: content/applications/websites/website/pages.rst [hunk 3]
```diff
@@ -1,9 +1,186 @@
...
+~~~~~~~~~~~~~~
+
+To delete a page, proceed as follows:
+
+#. Access the page, then go to :menuselection:`Website --> Site --> Properties` and click
+   :guilabel:`Delete Page`.
+#. A pop-up window appears on the screen with all links referring to the page you want to delete,
+   organized by category. Expand a category, e.g., :guilabel:`Page`, then click on a link to open
+   it in a new window.
+#. Once you have updated the links (or set up a :ref:`redirection <website/URL-redirection>`),
```
[the remark this answers is not in the document]
Comment (auva-odoo) [reply content/applications/websites/website/pages.rst:121]: You're right! I used your suggestion, but changed it slightly :)

File: content/applications/websites/website/pages.rst [hunk 4]
```diff
@@ -1,9 +1,186 @@
...
+   - :guilabel:`301 Moved Permanently`: for permanent redirections of static pages; the new URL is
+     shown in search engine results and the redirect is cached by browsers.
+   - :guilabel:`302 Moved Temporarily`: for short-term redirections, for example, if you are
+     redesigning or updating the page. The new URL is neither cached by browsers, nor shown in
+     search engine results.
+   - :guilabel:`308 Redirect / Rewrite`: for permanent redirections of existing Odoo routes/
+     controllers. Use this redirect type to rename a controller, for example, if you wish to rename
+     `/shop` into `/market`.
+
+- :guilabel:`URL from`: Enter the URL to be redirected (e.g., `/about-the-company`) or search for
```
[the remark this answers is not in the document]
Comment (auva-odoo) [reply content/applications/websites/website/pages.rst:164]: They are only listed when you select the 308 redirection type. We can have a look together :)

File: content/applications/websites/website/pages.rst [hunk 5]
```diff
@@ -1,9 +1,187 @@
...
+   organized by category. To ensure website visitors don't land on a 404 error page, you must update
+   all the links on your website referring to the page. To do so, expand a category, e.g.,
+   :guilabel:`Page`, then click on a link to open it in a new window.
+#. Once you have updated the links (or set up a :ref:`redirection <website/URL-redirection>`),
+   select the :guilabel:`I am sure about this` check box, then click :guilabel:`OK`.
+
+.. note::
+   Visitors are redirected to a 404 error page when they try to access a deleted page, either
+   directly or through a link that hasn't been updated. You can prevent this by defining a
+   :ref:`redirection <website/URL-redirection>` for the page's URL.
```
Comment (xpl-odoo) [inline content/applications/websites/website/pages.rst:128]:
```suggestion
```
Not necessary, given it's already covered by points 2. and 3.

File: content/applications/websites/website/pages.rst [hunk 6]
```diff
@@ -1,9 +1,187 @@
...
+URL redirects
+-------------
+
+Redirecting URLs consists in sending visitors and search engines to a URL that is different from the
+one they originally requested. This technique is used, for example, to prevent broken links when you
+:ref:`delete a page <website/delete-page>`, :ref:`modify its URL <website/page_properties>`, or move
+your site to a new :doc:`domain </administration/maintain/domain_names>`. It can also be used to
+improve :doc:`pages/seo`.
+
+To access existing URL redirections for your website and create new ones, :doc:`activate the
```
Comment (xpl-odoo) [inline content/applications/websites/website/pages.rst:141]:
```suggestion
To access existing URL redirections and create new ones, :doc:`activate the
```

File: content/applications/websites/website/pages.rst [hunk 7]
```diff
@@ -1,9 +1,187 @@
...
+developer mode </applications/general/developer_mode>` and go to :menuselection:`Website -->
+Configuration --> Redirects`.
+
+.. note::
+   A record is added automatically everytime you :ref:`modify a page's URL
```
Comment (xpl-odoo) [inline content/applications/websites/website/pages.rst:146]:
```suggestion
   A record is added automatically every time you :ref:`modify a page's URL
```

File: content/applications/websites/website/pages.rst [hunk 8]
```diff
@@ -1,9 +1,187 @@
...
+   select the :guilabel:`I am sure about this` check box, then click :guilabel:`OK`.
+
+.. note::
+   Visitors are redirected to a 404 error page when they try to access a deleted page, either
+   directly or through a link that hasn't been updated. You can prevent this by defining a
+   :ref:`redirection <website/URL-redirection>` for the page's URL.
+
+.. _website/URL-redirection:
+
+URL redirects
```
Comment (xpl-odoo) [inline content/applications/websites/website/pages.rst:132]: Does this apply to all page types (forum, blog, etc?)
If so, consider moving that part of the documentation to another page or specify this.

File: content/applications/websites/website/pages.rst [hunk 9]
```diff
@@ -1,9 +1,187 @@
...
+   - :guilabel:`308 Redirect / Rewrite`: for permanent redirections of existing Odoo routes/
+     controllers. Use this redirect type to rename a controller, for example, if you wish to rename
+     `/shop` into `/market`.
+
+- :guilabel:`URL from`: Enter the URL to be redirected (e.g., `/about-the-company`) or search for
+  the desired route/controller and select it from the list.
+- :guilabel:`URL to`: For 301, 202 and 308 redirects, enter the URL to be redirected to. If you want
+  to redirect to an external URL, make sure to include the protocol (e.g., `https://`).
+- :guilabel:`Website`: Select a specific website.
+- :guilabel:`Sequence`: to define the order in which redirections are performed, e.g. in the case of
```
Comment (xpl-odoo) [inline content/applications/websites/website/pages.rst:170]:
```suggestion
- :guilabel:`Sequence`: to define the order in which redirections are performed, e.g., in the case of
```

File: content/applications/websites/website/pages.rst [hunk 10]
```diff
@@ -1,9 +1,187 @@
...
+
+Toggle the :guilabel:`Activate` switch to deactivate the redirection.
+
+.. note::
+   - 301 and 308 redirections are cached by browsers.
+   - 404, 301 and 302 redirections only work if the original page has been :ref:`unpublished
+     <website/un-publish-page>` or :ref:`deleted <website/delete-page>`.
+
+.. seealso::
+   - `Redirects and Google Search <https://developers.google.com/search/docs/crawling-indexing/301-redirects>`_
```
Comment (xpl-odoo) [inline content/applications/websites/website/pages.rst:182]:
```suggestion
   - `Google documentation on redirects and search <https://developers.google.com/search/docs/crawling-indexing/301-redirects>`_
```

**Review by xpl-odoo — CHANGES_REQUESTED** [review]
Nice job Audrey/Tom! :) Don't hesitate to ask questions regarding my suggestions or challenge them ;)

**Review by xpl-odoo — APPROVED** [review]
@robodoo delegate+

**Comment by auva-odoo** [comment]
@robodoo r+

**Comment by xpl-odoo** [comment]
@robodoo r+

**Pull request closed without merging**